plugins/modules/lvg.py

```diff
@@ -156,6 +200,176 @@ def parse_pvs(module, data):
     return pvs
 
 
+def find_vg(module, vg):
+    vgs_cmd = module.get_bin_path('vgs', True)
```

Comment:
I think you can do a quick exit if `not vg`:
```suggestion
    if not vg:
        return None
    vgs_cmd = module.get_bin_path('vgs', True)
```